### PR TITLE
Move django project to repository root

### PR DESCRIPTION
**Pull request by jegorami**
Fix licence and README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
-# living-environment
\ No newline at end of file
+# Living Environment
+
+This learning platform aims to inform young citizens about the maintenance of the urban city environment, as well as show them the way to influence their environment.
```
Comment (JsseL) [inline README.md:3]: Well said.

**Review by JsseL — APPROVED** [review]
Ah, good thing you fixed this. Missed the incorrect dir structure in #1. Good job :+1:

**Comment by jegorami** [comment]
@arvileino would you please merge it to master?

**Comment by JsseL** [comment]
@arvileino will merge it when he has reviewed it.

**Edit:** Sorry, read incorrectly. I thought you were asking me to merge.. :raised_hand_with_fingers_splayed: my bad.

**Pull request merged**